src/main/java/org/fundacionjala/sfdc/pageobjects/accounts/SFANewModifyPage.java

```diff
@@ -0,0 +1,128 @@
+package org.fundacionjala.sfdc.pageobjects.accounts;
+
+import org.fundacionjala.sfdc.util.CommonActions;
+import org.fundacionjala.sfdc.util.Value;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * SFANewModifyPage.java
+ * Class that represents the form where you fill fields in order to create or modify an account.
+ */
+public class SFANewModifyPage extends SalesForceConnection{
+    private Map<String, Value> NEW_ACCOUNTS_FIELDS = new HashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "NEW_ACCOUNTS_FIELDS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)